projects/cashmere/src/lib/navbar/navbar-mobile-menu/navbar-mobile-menu.component.ts

```diff
@@ -29,6 +27,6 @@ export class NavbarMobileMenuComponent {
 
     /** Hide the component from view */
     hide(): void {
-        this._yPos = '-100';
+        this._yPos = '-50';
```

Comment:
any reason this isn't being removed or display:none?

Reply:
It's for the animation to slide up/down